lib/node_modules/@stdlib/math/base/special/log10/src/main.c

```diff
@@ -90,31 +91,33 @@ double stdlib_base_log10( const double x ) {
 	}
 	xc = x;
 	stdlib_base_float64_to_words( xc, &hx, &lx );
+	ihx = ( int32_t )hx;
```

Comment:
```suggestion
	ihx = (int32_t)hx;
```

For casts, we don't include spaces.